src/pages/components/tile/usage.mdx

```diff
@@ -276,44 +299,51 @@ modules. Here are some common use cases for when to use tiles:
 
 ## Tile variants
 
+<InlineNotification>
+
+Except for base tiles, experimental tiles are applicable for clickable,
+selectable, and expandable tiles.
+
+</InlineNotification>
+
 Tiles can function or be implemented in four ways—base, clickable, selectable,
 or expandable. These variations of tiles are flexible enough to support a
-variety of different use cases when building more complicating card features.
+variety of different use cases when building more complicated card features.
 
 | Variant                   | Purpose                                                                                                  |
 | ------------------------- | -------------------------------------------------------------------------------------------------------- |
 | [Base](#base)             | For high-level, short, and digestible content pieces such as features, plans, or services offered        |
 | [Clickable](#clickable)   | For prompting an action, navigating or directing to other pieces of information about the subject matter |
 | [Selectable](#selectable) | For presenting options to a user in a structured manner, such as a set of pricing plans                  |
-| [Expandable](#expandable) | For hiding and revealing a large amount of content in order to focus on specific pieces of info          |
+| [Expandable](#expandable) | For hiding and revealing a large amount of content to focus on specific pieces of information            |
 
 ### Base
 
 Base tiles are used to display information to the user, such as features or
 services offered. Base tiles are often seen on marketing pages to promote
-content or on dashboards that are highly interactive. These tiles can have
-internal calls-to-action (CTAs), such as a [button](/components/button/usage) or
-a [link](/components/link/usage).
+content or on highly interactive dashboards. These tiles can have internal
+calls-to-action (CTAs), such as a [button](/components/button/usage) or a
+[link](/components/link/usage).
 
 <Row>
-<Column colLg={12}>
+<Column colLg={8}>
 
-![Example image of base tiles.](images/tile-usage-1.png)
+![Example image of base tiles](images/tile-usage-2.png)
 
 </Column>
 </Row>
 
 ### Clickable
 
-Clickable tiles can be used as navigational items, where the entire tile is a
+Clickable tiles can be used as navigational items, where the entire tile is in a
```

Comment:
Clickable tiles can be used as navigational elements that redirect the user to a new page. In these situations, the entire tile is in a clickable state. Due to accessibility concerns, clickable tiles cannot contain separate internal CTAs but can contain pictograms, icons, or media such as illustrations or images.